### DISCUSSION TITLE
Integration der csv-Dateien in ebusd

### DISCUSSION THREAD

**Discussion by solarpirats** [cyberthom42/vaillant-arotherm-plus#25] [Q&A]
Hallo @cyberthom42,
habe jetzt das repo kopiert und versuche die csv in ebusd zu laden. Anscheinend funktioniert bei mir die _templates.csv nicht, kannst Du eine kurze Anleitung schreiben, welche Datei wo stehen muss? Ich weiß wie ich den config-Pfad einstelle, funktioniert auch aber die csv werden nicht erkannt. Mein System hat die gleiche Konfiguration wie meierchen006 im hausbau-forum. Bei ihm geht es, bei mir nicht...
Danke!
Frank

version: ebusd 24.1.24.1
update check: OK, device firmware v5 1[4a1b] up to date
device: 192.168.179.9:9999, TCP, enhanced, firmware 1.1[4a1b].1[4a1b]
signal: acquired
symbol rate: 95
max symbol rate: 181
min arbitration micros: 7
max arbitration micros: 39
min symbol latency: 5
max symbol latency: 45
scan: finished
reconnects: 0
masters: 5
messages: 14
conditional: 0
poll: 2
update: 0
address 03: master #11
address 08: slave #11, scanned "MF=Vaillant;ID=HMU00;SW=0902;HW=5103"
address 10: master #2
address 15: slave #2, scanned "MF=Vaillant;ID=CTLV3;SW=0808;HW=8004"
address 26: slave, scanned "MF=Vaillant;ID=VR_71;SW=0201;HW=0503"
address 31: master #8, ebusd
address 36: slave #8, ebusd
address 71: master #9
address 76: slave #9, scanned "MF=Vaillant;ID=VWZIO;SW=0902;HW=5103"
address f1: master #10
address f6: slave #10, scanned "MF=Vaillant;ID=NETX2;SW=4035;HW=5703"

**Comment by cyberthom42** [comment]
Du musst die Dateien anpassen. Z.B. habe ich eine Sensocomfort mit Funk, die meldet sich als basv. Bei dir ist es ctlv3. Du musst dann entweder einene Link erstellen oder die basv umbenennen in ctlv3. Gleiches gilt wahrscheinlich für HMU und andere
